client_test.go

```diff
@@ -312,6 +315,9 @@ func TestBeforeSendGetAccessToEventHint(t *testing.T) {
 	client, scope, transport := setupClientTest()
 	client.options.BeforeSend = func(event *Event, hint *EventHint) *Event {
 		if ex, ok := hint.OriginalException.(customComplexError); ok {
+			if len(event.Exception) != 1 {
```

Comment:
`!= 1` and `missing` are logically not correct, as `x == 2` is not missing, yet it'd not pass :)

Reply:
You're right.

In the newer tests we have we check for the whole "exception" slice -- so it is clear the number of items we expect in the slice.

`if len(...) > 0` would be sufficient to work around the panic in the next line. But the intention with `!= 1` is to guard also against wrong number of events.

Best is to rewrite these tests.

Will iterate on this later.